### ISSUE TITLE
Add optional email-notification when a IP-adress is blocked by brute-force-protection

### ISSUE THREAD

**Issue by criwe** [nextcloud/server#2356]
Hi all!

It would be nice, to have the option to get an email notification when an IP-adress is blocked by brute-force-protection. 
Its good to know when an attack occurs, so you can i.e. add some firewall or htaccess rules to block some aggressive IPs completely.
Best,
Chris

**Comment by MorrisJobke** [comment]
@karlitschek @LukasReschke Should we build in this or should this be handled by external tools (monitoring, fail2ban, etc...)

**Comment by MorrisJobke** [comment]
> @karlitschek @LukasReschke Should we build in this or should this be handled by external tools (monitoring, fail2ban, etc...)

I tend to use existing tools here.

**Comment by nickvergessen** [comment]
Well so the next thing we do is help to DDoS your mail provider?

Maybe should log this list somehow, but definitely not send emails ourselves.

**Comment by karlitschek** [comment]
I agree. A mail notification is out of scope for Nextcloud. But we should write it to a logfile so that this can be processed by the right system monitoring and administration tools.

**Comment by MorrisJobke** [comment]
It is already logged: https://github.com/nextcloud/server/blob/6c93fe08f53bff474921d150edabb27ca630edd7/lib/private/Security/Bruteforce/Throttler.php#L171-L171 (but on notice/info level)

**Comment by nickvergessen** [comment]
So this can already be done by log readers, closing
